pyproject.toml

```diff
@@ -13,6 +13,10 @@ Werkzeug = "^2.0.1"
 MarkupSafe = "^2.0.1"
 selenium = "^3.141.0"
 gunicorn = "^20.1.0"
+pytest = "^6.2.5"
+watchdog = "^2.1.5"
+PyYAML = "^5.4.1"
+argh = "^0.26.2"
```

Comment:
Minor, but worth noting that these are all only being used in development, so they're not necessary for production, but `poetry install` will pick these up by default. We could move them into the `dev-dependencies` section below (manually, or originally by adding them with `poetry add -D <package_name>`) and we could then add the `--no-dev` flag to our production `poetry install ...` command